diff --git a/sources/PseudoTerminal.m b/sources/PseudoTerminal.m
index f6868f3..99cded2 100644
--- a/sources/PseudoTerminal.m
+++ b/sources/PseudoTerminal.m
@@ -354,7 +354,6 @@ static NSString* TERMINAL_ARRANGEMENT_HIDING_TOOLBELT_SHOULD_RESIZE_WINDOW = @"H
 
         default:
             return (mask |
-                    NSTitledWindowMask |
                     NSClosableWindowMask |
                     NSMiniaturizableWindowMask |
                     NSResizableWindowMask |
diff --git a/sources/iTermTextDrawingHelper.h b/sources/iTermTextDrawingHelper.h
index e6ab4c6..bde883d 100644
--- a/sources/iTermTextDrawingHelper.h
+++ b/sources/iTermTextDrawingHelper.h
@@ -13,10 +13,10 @@
 #import "VT100GridTypes.h"
 
 // Number of pixels margin on left and right edge.
-#define MARGIN 5
+#define MARGIN HORIZONTAL_VAR
 
 // Number of pixels margin on the top.
-#define VMARGIN 2
+#define VMARGIN VERTICAL_VAR
 
 @class iTermColorMap;
 @class iTermFindOnPageHelper;
